flow/scripts/mem_dump.py

```diff
@@ -0,0 +1,29 @@
+import json
+import sys
+
+
+def format_ram_table_from_json(data):
+    table = "{:<20} | {:<15} | {:<15} | {:<15}\n".format("Module Name",
```

Comment:
store the format string once

Reply:
That would pollute the global namespace for very little performance gain and could break if some dynamic strings later are added to the header.

Reply:
Why would it be global -its repeated twice in the same function

Reply:
OIC.

Reply:
Fixed.